webapp-django/crashstats/signature/static/signature/css/signature_report.less

```diff
@@ -118,7 +119,19 @@ a.crash-id {
     }
 }
 
-#graph-panel {
+.graph-panel {
+    .panel {
+        .options {
```

Comment:
Good point, I guess it's a historical artefact. Actually would make more sense to have a panel-within-tab class so this doesn't need to be repeated for every tab with panels.